package.json

```diff
@@ -101,14 +100,13 @@
     }
   },
   "dependencies": {
-    "@angular/animations": "9.1.1",
```

Comment:
Why downgrade Angular? Libraries can be built with Angular 9 and still work with `ViewEngine`

Reply:
When ivy is enabled (with ngcc v8 + ng 9) for apps they crash (see below) because some compiler symbols form ng 9 are not found. So I decided to downgrade the angular libs, to get a consistent behaviour across possible configurations.

![image](https://user-images.githubusercontent.com/107658/85848600-b7d88d80-b7a9-11ea-9bd4-8b64b563ccfc.png)